### PR TITLE
Add UI tests for StaffCard and IngredientCard

### PR DESCRIPTION
**Pull request by zj-cs2103**
Will follow up with tests for panes.

### CODE REVIEW

File: src/test/java/guitests/guihandles/IngredientCardHandle.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    public String getUnit() {
+        return unitLabel.getText();
+    }
+
+    public String getQuantity() {
+        return quantityLabel.getText();
+    }
+
+    /**
+     * Returns true if this handle contains {@code staff}.
```
Comment (lingyanhao) [inline src/test/java/guitests/guihandles/IngredientCardHandle.java]: change this javadoc

File: src/test/java/seedu/address/ui/IngredientCardTest.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        Ingredient differentIngredient = new IngredientBuilder()
+                .withIngredient("differentName", 1, "differentUnit").build();
+        assertFalse(ingredientCard.equals(new IngredientCard(differentIngredient, 0)));
+
+        // same ingredient, different index -> returns false
+        assertFalse(ingredientCard.equals(new IngredientCard(ingredient, 1)));
+    }
+
+    /**
+     * Asserts that {@code staffCard} displays the details of {@code expectedIngredient} correctly and matches
```
Comment (lingyanhao) [inline src/test/java/seedu/address/ui/IngredientCardTest.java]: change the javadoc, staffCard -> ingredientCArd

File: src/test/java/seedu/address/ui/IngredientCardTest.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+     */
+    private void assertCardDisplay(IngredientCard ingredientCard, Ingredient expectedIngredient, int expectedId) {
+        guiRobot.pauseForHuman();
+
+        IngredientCardHandle ingredientCardHandle = new IngredientCardHandle(ingredientCard.getRoot());
+
+        // verify id is displayed correctly
+        assertEquals(Integer.toString(expectedId) + ". ", ingredientCardHandle.getId());
+
+        // verify staff details are displayed correctly
```
Comment (lingyanhao) [inline src/test/java/seedu/address/ui/IngredientCardTest.java]: ingredient details

File: src/test/java/seedu/address/ui/MemberCardTest.java [hunk 4]
```diff
@@ -35,7 +35,7 @@ public void equals() {
         assertTrue(memberCard.equals(memberCard));
 
         // null -> returns false
-        assertFalse(memberCard.equals(null));
+        assertFalse(memberCard == null);
```
Comment (lingyanhao) [inline src/test/java/seedu/address/ui/MemberCardTest.java]: I think the `.equals()` here is correct- it is meant to check if the `.equals() ` function works correctly on `null` objects.

File: src/test/java/guitests/guihandles/IngredientCardHandle.java (hunk 1 above)
Comment (zj-cs2103) [reply src/test/java/guitests/guihandles/IngredientCardHandle.java]: fixed

File: src/test/java/seedu/address/ui/IngredientCardTest.java (hunk 2 above)
Comment (zj-cs2103) [reply src/test/java/seedu/address/ui/IngredientCardTest.java]: fixed

File: src/test/java/seedu/address/ui/IngredientCardTest.java (hunk 3 above)
Comment (zj-cs2103) [reply src/test/java/seedu/address/ui/IngredientCardTest.java]: fixed

File: src/test/java/seedu/address/ui/MemberCardTest.java (hunk 4 above)
Comment (zj-cs2103) [reply src/test/java/seedu/address/ui/MemberCardTest.java]: fixed

**Pull request merged**